gke-deploy.sh

```diff
@@ -17,6 +17,10 @@
 
```

Comment:
Are there any preconditions to complete before running the script? You can add to comments or README if there are.